package.json

```diff
@@ -44,8 +49,11 @@
         "webpack": "^4.39.1"
     },
     "dependencies": {
+        "assert": "^2.0.0",
         "bcrypto": "^4.2.6",
+        "bip39": "^3.0.2",
         "bn.js": "^5.0.0",
-        "buffer": "^5.4.3"
+        "buffer": "^5.4.3",
+        "secure-random": "^1.1.2"
```

Comment:
By the way, there is a `bcrypto` module `bcrypto/lib/random` for generating random numbers. It binds to OpenSSL using `"openssl/rand.h"`
https://github.com/bcoin-org/bcrypto/blob/master/src/random/random.c

It also runs in browsers:
https://github.com/bcoin-org/bcrypto/blob/master/lib/js/random.js

See https://github.com/ChainSafe/bls-ts-key-mgmt/issues/3